kivy/graphics/cgl.pxd

```diff
@@ -639,6 +675,8 @@ ctypedef struct GLES2_Context:
     void (__stdcall *glVertexAttribPointer)(GLuint, GLint, GLenum, GLboolean, GLsizei, const void *) nogil
     void (__stdcall *glViewport)(GLint, GLint, GLsizei, GLsizei) nogil
 
+    GLX_Context *glx
```

Comment:
I'm not really sure, but if we use GLES2 on mobile devices, is this a good approach? I mean, does it compile on phones if this change is present?

Reply:
You are correct, this breaks the build on non-Linux/x11 systems, and it's something that needs changed. I haven't figured out a good solution yet, because the window provider, graphics API and display server all change the implementation of TFP. Some platforms (Android/iOS) likely can't support it in any form.